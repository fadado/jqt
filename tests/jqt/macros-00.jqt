<# define required partials #>&
<%shortcode youtube>&
<%shortcode analytics>&
<# expand partials #>&
<%youtube 7zIoLvbCCm8 420 315>
<%analytics UA-8673551-2>
