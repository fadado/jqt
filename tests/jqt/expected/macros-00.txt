<iframe src="https://www.youtube.com/embed/7zIoLvbCCm8"
        width="420"
        height="315"
        frameborder="0"></iframe>

<script type="text/javascript">/*<![CDATA[*/
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-8673551-2', 'auto');
  ga('send', 'pageview');
/*]]>*/</script>


